### PR TITLE
Using Bootstrap

### PR DESCRIPTION
**Pull request by afcastaneda223**
- Using Bootstrap to Clone Newsweek page
- Added layout using row and col properties from bootstrap
- Added images and text taken from the Newsweek page

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @afcastaneda223 
### Good job for this far you have come :clap: However, there are changes you need to make.
- Write a more descriptive pull request [check this guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- In the top story section, make the height of the content to equal content on the left and right by defining its height and making it scrollable, repeat this on other screens too
overlapping
- On mobile screen, your content is going full width, add some padding or margin, make it same with the other content, refer image below.
mobile
- Avoid too much nesting in your html code, you can achieve this in bootstrap easily by defining row and columns
nesting

- Make these changes and resubmit for a code review. I wish you all the best
- Happy coding!
- For assistance: Contact me on @rOluochKe

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great job @afcastaneda223 !
You have done really nicely and your project is coming together nicely! :clap: Please consider my comments below, make the necessary adjustments and request another review :100: I wish you all the best :smile: 

#### Navigation bar
Your navigation bar is a little buggy, the positioning of elements, and padding of the navigation are in a little twist. Consider comparing what the original site looks like(including but not limited to responsiveness) to yours and make any appropriate changes. 

##### What your navbar looks like on mobile
screenshot

#### [OPTIONAL] Scrollbar
Please consider removing the scroll bar on the center section of the page, I understand the original has some sort of sticky effect and venturing into that or learning about it is something I'll encourage, however, it isn't necessary to replicate the effect and even more so if it will take some polish off your project. Please consider removing the overflow properties you have set on that section, let the full height show, it is completely okay.
screenshot


#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

##### What's happening
screenshot

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @afcastaneda223, you are getting there:+1:. Let's address the following:**
1. Our navbar should fill at least 95% of the header width.
image

2. Give article titles a bold font weight.

3. Reduce font-sizes for these sections:
image
The current font-size seems too big when compared with the original site and it causes a distortion on the right-most column at certain widths on medium screen.

4. You have one html validation error that has cropped up. Check it and fix it.

_**Kindly make these adjustments and submit another code review request.**_

**Cheers:stars:**

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1747 @@
...
+                                </div>
+                            </div>
+                            <div class="col-sm-12 col-md-6 col-lg-3 m-0 p-0 pl-md-2 px-lg-1">
+                                <div>
+                                    <figure class="figure">
+                                        <img src="https://d.newsweek.com/en/full/1566006/oscar-statuette.jpg?"
+                                            class="img-fluid" alt="Oscar statuette ">
+                                    </figure>
+                                    <h4>
+                                        <a href="#">Goody B
```
Comment (addod19) [inline index.html]: Kindly wrap all ```<li>``` with ```<a>```
newlinks

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @afcastaneda223, 😄 you did a great job implementing some of the changes requested by the previous reviewers. 💯  Your project is very responsive 👍 , However, we still have some requirements left out. Find comments below:
- The nav should look like the original since it is a clone. Currently, this is how the nav looks like. Consider the extra spacing behind the search button
newnav
```Original```
newnav1
- The footer should have links, currently, we do not have links in the footer.
[OPTIONAL]
```Tablet mode``` Currently, in Culture and travel this is how it looks
cul
```Original```
cul1

Kindly make these changes and submit another code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @afcastaneda223       
#### Great Job so far, but there is one more thing required. 
### General
- [ ] Lets give the nav links **Padding** at the **Top**


.



 

### Make changes and submit another code review request.
*Happy coding*

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**